core/Cargo.toml

```diff
@@ -47,6 +47,8 @@ libp2p-tcp = { path = "../transports/tcp" }
 multihash = { version = "0.13", default-features = false, features = ["arb"] }
 quickcheck = "0.9.0"
 wasm-timer = "0.2"
+# TODO: For testing only. Remove.
```

Comment:
Will remove this before merging. Leaving it for now to ensure I don't introduce a regression.